contrib/grinder/scripts/ingest.py

```diff
@@ -93,7 +93,8 @@ def make_request(self, logger):
                                         self.slice[self.position])
         self.position += 1
         result = self.request.POST(self.ingest_url(), payload)
+        if result.getStatusCode() == 500:
```

Comment:
@shintasmith @usnavi This code is moved to a new repo now right? We have to remove the grinder folder from blueflood or retrofit new changes to new repo till we remove this folder from blueflood.